### PR TITLE
Add create garden + rules

### PR DESCRIPTION
**Pull request by kome12**
## Description

Add create garden + rules

## Closes issue(s)

#26 
#40 

## Screenshots

If you have image share...

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code

### CODE REVIEW

File: src/pages/CreateGarden/index.tsx [hunk 1]
```diff
@@ -38,82 +45,130 @@ export const CreateGarden = ()  => {
...
 
   const createGardenHandler = (e: React.MouseEvent<HTMLButtonElement>) => {
-    console.log("Make API call to create garden here.")
-  }
+    const createGardenAndRules = async () => {
+      // TODO: FIX the endpoint
+      const newGarden: Garden = {
+        name: name,
+        description: desc,
+        userFireBaseId: userData?.id || "",
```
Comment (conalli) [inline src/pages/CreateGarden/index.tsx:80]: Is it possible that we wouldn't have the id at this point?

File: src/pages/CreateGarden/index.tsx [hunk 2]
```diff
@@ -38,82 +45,130 @@ export const CreateGarden = ()  => {
...
     setRuleName("");
     setRuleDesc("");
-  }
+  };
 
   const createGardenHandler = (e: React.MouseEvent<HTMLButtonElement>) => {
-    console.log("Make API call to create garden here.")
-  }
+    const createGardenAndRules = async () => {
+      // TODO: FIX the endpoint
```
Comment (conalli) [inline src/pages/CreateGarden/index.tsx:76]: Is this todo referencing the useApi that Daichi is making?

**Review by conalli — APPROVED** [review]
LGTM

File: src/pages/CreateGarden/index.tsx (hunk 1 above)
Comment (kome12) [reply src/pages/CreateGarden/index.tsx:80]: yeah, it's possible so next week we need to better manage this.

File: src/pages/CreateGarden/index.tsx (hunk 2 above)
Comment (kome12) [reply src/pages/CreateGarden/index.tsx:76]: Yup, we shouldn't call axios every time so next week can you update this?

**Pull request merged**